app/ts/types/JsonRpc-types.ts

```diff
@@ -402,6 +408,7 @@ export const EthereumJsonRpcRequest = funtypes.Union(
 	UninstallFilter,
 	GetFilterChanges,
 	GetFilterLogs,
+	EthChainId,
```

Comment:
this aleady existst above:
`funtypes.ReadonlyObject({ method: funtypes.Literal('eth_chainId') }),`, remove that if you need to have it separated.